src/GovUk.Education.ExploreEducationStatistics.Admin/Models/ArchiveDataSetFile.cs

```diff
@@ -1,9 +1,12 @@
 #nullable enable
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Models;
 
 public record ArchiveDataSetFile(
     string Title,
     string DataFilename,
     string MetaFilename,
     long DataFileSize = 1048576,
-    long MetaFileSize = 1024);
+    long MetaFileSize = 1024,
```

Comment:
Appreciate you didn't add these, but what's the purpose of the default values here?

Reply:
Ben advised that this may be to avoid repeating values in unit tests, and recommended creating a test data generator instead - made a note to create one and remove these defaults in my current branch.